### PR TITLE
A blog post about our near term plans.

### PR DESCRIPTION
**Pull request by anuveyatsu**
This PR fixes https://github.com/datopian/datahub/issues/1322

### CODE REVIEW

File: what-is-next-enhancing-dataset-discovery-and-providing-core-data-for-the-world.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+In the world of data, accessibility and quality are crucial. As we move into the next phase of DataHub.io, our goal is to make it the go-to place for finding core and popular datasets. Alongside that, we're building a seamless experience for data publishers to upload and showcase their datasets. Over time, we envision this evolving into a vibrant data marketplace.
+
+## Our Focus for the Next 3-6 Months
+
+To realize this vision, we are dedicating the next few months to several key improvements:
+
+### Enhanced Dataset Showcase Pages
+
+We’re transforming dataset pages into rich, informative showcases. Our goal is for users to quickly grasp the dataset’s metadata, preview the data, and understand the schema in detail. We’ll offer comprehensive insights into the structure and contents of each dataset, empowering users to make informed decisions.
```
Comment (olayway) [inline what-is-next-enhancing-dataset-discovery-and-providing-core-data-for-the-world.md:15]:
```suggestion
We’re transforming dataset pages into rich, informative showcases. Our goal is for users to quickly grasp a dataset’s metadata, preview its data, and understand its schema in detail. We’ll provide comprehensive insights into the structure and contents of each dataset, empowering users to make informed decisions.
```

**Review by olayway — APPROVED** [review]
Looks awesome! Just two very minor suggestions.

File: what-is-next-enhancing-dataset-discovery-and-providing-core-data-for-the-world.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+title: "What is next: Enhancing Dataset Discovery and Providing Core Data for the World"
+date: 2024-09-27
+authors: [Anuar Ustayev]
+---
+
+In the world of data, accessibility and quality are crucial. As we move into the next phase of DataHub.io, our goal is to make it the go-to place for finding core and popular datasets. Alongside that, we're building a seamless experience for data publishers to upload and showcase their datasets. Over time, we envision this evolving into a vibrant data marketplace.
```
[the remark this answers is not in the document]
Comment (anuveyatsu) [reply what-is-next-enhancing-dataset-discovery-and-providing-core-data-for-the-world.md:7]: i like "essential" 👍🏼

**Comment by anuveyatsu** [comment]
I resolved your feedback @olayway and now merging this 👍🏼

**Pull request merged**